registry.json

```diff
@@ -3372,5 +3372,20 @@
     "telegram": "DeCentN2Madness",
     "operatorName": "Jeff Hales",
     "github": "qbass"
+  },
+  {
+    "ticker": "STEL",
+    "poolId": "31936b317654f36cde59e4c23a04b92e6565b8d95f261b74e94c823f",
+    "social": {
+      "facebook": "Stellence",
```

Comment:
I've checked all these socials - there is very little content and nothing about the pool/cardano/ada... Please can you add some content?